src/styles/blocks/catalog.scss

```diff
@@ -11,15 +11,21 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition: 0.5s;
```

Comment:
You need to specify for which properties the transitions should work
https://developer.mozilla.org/en-US/docs/Web/CSS/transition